.eslintrc.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+    "globals": {
+        "document": true,
+        "window": true,
+        "MutationObserver": true,
+        "requestIdleCallback": true,
```

Comment:
Aren't these four + `fetch` included in the `webextensions` env?

Reply:
No, I still get no-undef for them.

Reply:
Let's add the `browser` env too then, they should be defined in that one.

Reply:
Actually, the `browser` env might be implied automatically if we use the mozilla plugin's rules.